Assets/Scripts/WizardMovement.cs

```diff
@@ -19,6 +19,12 @@ public class WizardMovement : MonoBehaviour
     //----------------------------------------
     public bool updated;
 
+    public string keyUp = "player1_move_up";
```

Comment:
Again, maybe use some globally available enumeration?
You have done this for much smaller value collections too (AggroType).

Then, the "configuration keys" for the bindings can be found in one place
and you have a restricted set of known values (which is always good, because
maybe you accidentally misspell something and try to use a value that does not exist.